cores/esp8266/WString.cpp

```diff
@@ -411,8 +413,9 @@ bool String::concat(const __FlashStringHelper *str) {
     unsigned int newlen = len() + length;
     if (!reserve(newlen))
         return false;
-    memcpy_P(wbuffer() + len(), (PGM_P)str, length + 1);
+    memcpy_P(wbuffer() + len(), (PGM_P)str, length);
```

Comment:
`strlen_P()` is called on this one,
`\0` presence is then ensured ?

Reply:
You are right in this case. I can revert it back, I just ended up replacing everywhere that copied length + 1 as it seemed error prone.

Reply:
That's OK, no need for a further update :+1: